packages/rebber-plugins/src/preprocessors/mathEscape.js

```diff
@@ -1,6 +1,6 @@
 const katexConstants = require('../../src/preprocessors/katexConstants.json')
 
-const endOfCommandChars = ['\\', '[', ']', '{', '}', ' ', '(', ')', '\t', '\n']
+const endOfCommandChars = ['\\', '[', ']', '{', '}', ' ', '(', ')', '\t', '\n', '^', '_', ',']
```

Comment:
This array seems to just grow bigger and bigger :smile: 

If we have one more problem, I will find another way to detect end of command.